subprojects/docs/src/docs/release/notes.md

```diff
@@ -456,24 +457,12 @@ ADD RELEASE FEATURES ABOVE
 -->
 
 ## Promoted features
+
 Promoted features are features that were incubating in previous versions of Gradle but are now supported and subject to backwards compatibility.
 See the User Manual section on the “[Feature Lifecycle](userguide/feature_lifecycle.html)” for more information.
 
 The following are the features that have been promoted in this Gradle release.
 
-### Replacement collections in `org.gradle.plugins.ide.idea.model.IdeaModule`
-
-The `testResourcesDirs` and `testSourcesDirs` fields, and their getters and setters are now `@Deprecated`.
-Any usages of these elements should be replaced by the now stable `getTestSources()` and `getTestResources()` methods and their respective setters.
-These new methods return and are backed by `ConfigurableFileCollection` instances for improved flexibility in how these collections of files can be used.
-Gradle now warns upon usage of these deprecated methods that they will be removed in Gradle 8.0.
-
-### Replacement methods in `org.gradle.api.tasks.testing.TestReport`
-
-The `getDestinationDir()`, `setDestinationDir(File)`, and `getTestResultsDirs()` and `setTestResultsDirs(Iterable)` methods are now `@Deprecated`.
-Any usages of them should be replaced by the now stable `getDestinationDirectory()` and `getTestResults()` methods and their associated setters.
-These deprecated elements will be removed in Gradle 8.0.
-
```

Comment:
We still need part of this.

`getTestSources` and `getTestResources` are stable/promoted now.
`getDestinationDirectory` and `getTestResults` are stable/promoted now.

Reply:
I like the old usage with the new but as long as this is in the migration guide I am okay with removing.

Reply:
Added these back to promotions.